thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -1284,44 +1272,26 @@ export default Controller.extend({
         finalConfigObj.emailConfig.functionIds.push(newFunctionId);
 
         // Finally, save our Alert Config Groupg
-        this.saveThirdEyeEntity(finalConfigObj, 'ALERT_CONFIG').then(alertResult => {
-          // Display success confirmations including new alert Id and recipients
-          this.setProperties({
-            selectedGroupRecipients: finalConfigObj.recipients.replace(/,+/g, ', '),
-            isCreateAlertSuccess: true,
-            finalFunctionId: newFunctionId
-          });
-          // Confirm group creation if not in group edit mode
-          if (!isEditGroupMode) {
-            this.set('isCreateGroupSuccess', true);
-          }
-          // Display function added to group confirmation
-          this.prepareFunctions(finalConfigObj, newFunctionId).then(functionData => {
-            this.set('selectedGroupFunctions', functionData);
-          });
-
-          // Now, disable form
-          this.setProperties({
-            isFormDisabled: true,
-            isMetricSelected: false,
-            isMetricDataInvalid: false
-          });
-          this.set('newFuncId', newFunctionId);
-          return this.triggerReplay(newFunctionId);
-        }).then((replayId) => {
-
-          if (!this.get('isReplayStatusError')) {
+        this.saveThirdEyeEntity(finalConfigObj, 'ALERT_CONFIG')
+          .then(alertResult => {
+            // Display success confirmations including new alert Id and recipients
             this.setProperties({
-              isReplayStatusSuccess: true,
-              isReplayStatusPending: false,
-              replayStatusClass: 'te-form__banner--success'
+              selectedGroupRecipients: finalConfigObj.recipients.replace(/,+/g, ', '),
+              finalFunctionId: newFunctionId
             });
-          }
-
-          // Redirect to onboarding page to trigger wrapper
-          if (redirectToAlertPage) {
-            this.transitionToRoute('manage.alert', this.get('newFuncId'), { queryParams: { replayId }});
-          }
+            // Display function added to group confirmation
+            this.prepareFunctions(finalConfigObj, newFunctionId).then(functionData => {
+              this.set('selectedGroupFunctions', functionData);
+            });
+            // Now, disable form
+            this.setProperties({
```

Comment:
should `this.setProperties` and the `this.send` below happen after the this.preparefunctions's chained promises? if yes, the current implementation doesn't guarantee that

Reply:
Please disregard - I've removed everything except `this.send`.